Fall-2017-JS02/Challenge4.html

```diff
@@ -0,0 +1,62 @@
+<html>
+<body>
+  <script src = "https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  <h2>What is your name? <label class="msg"></label></h2>
+  <div style="padding:16px;">
+    Name : <input class='name' type="textbox" value="Type Name"></input>
+  </div>
+
+  <button class="Get">Get TextBox Value</button>
+
+  <script type="text/javascript">
+  $('button').click(function () {
+    $('.msg').html($('input.name').val());
+
+    console.log("Hi");
+
+  });
+  </script>
```

Comment:
This whole `<script>` block should be contained in your .js file. Also, there are a couple of small suggested changes: 

**Use the class that you specified**
I noticed in line 12 that you reference the button, but don't indicate which button you meant, though you did give the button its own class (`<button class="Get">`). On the click function, you instead would use `$('button.Get').click()`

**Use ES6 formats for functions**
There is a new way to specify a function in the new version of JavaScript (ECMAScript 6, or ES6). It used to be that functions were defined like this:

``` javascript
//define a  function 
function test() { 
   console.log("function called") 
} 

//call the function 
test()
```

This is now expressed as

```javascript
//define a  function 
const test = () => { 
   console.log("function called") 
} 

//call the function 
test() 
```

If the function is only used one time (like it is when you put it inside of the `.click()` function, you don't even need to give it a name:

``` javascript
$('.myButton').click( () => { 
   console.log("function called") 
} 
```

### What's the result?

Lines 11-18 would be deleted, and instead, in your .js file, you would include:

``` javascript
$('button.Get').click( () => {
   $('.msg').html($('input.name').val());
   console.log("Hi");
});
```


**Don't forget**
You will need to include a reference to your javascript file in your HTML file, usually at the very end of the `<body>` block. It will look something like this:

``` html
<script type="text/javascript" src="myJavaScript.js"></script>
```